gdraw/gradio.c

```diff
@@ -367,7 +367,7 @@ return( false );
             pts[i].y=.5*h*sin(angle)+y+h/4;
 
              /* draw lashes */
-            if (i>0 && i<5) GDrawDrawLine(pixmap, pts[i].x,pts[i].y, .75*w*cos(angle)+x+w/2, .75*h*sin(angle)+y+h/4, fg);
+            if (i>0 && i<6) GDrawDrawLine(pixmap, pts[i].x,pts[i].y, .75*w*cos(angle)+x+w/2, .75*h*sin(angle)+y+h/4, fg);
             ++i;
```

Comment:
I would favor moving this `++i` after `c++` (and a comma) in the for loop, as that is the logic, they're initialized together, and having the iteration separate is confusing.

Reply:
Done in https://github.com/fontforge/fontforge/pull/3931/commits/772f31dfd5d8c5b437b36c28f4d4f4e66da2600a